### PR TITLE
V3.1.0 Readme and Changelog

### PR DESCRIPTION
**Pull request by Cx01N**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,9 +62,9 @@ cd Empire
 sudo ./setup/install.sh
 ```
 
-There's also a [quickstart here](http://www.powershellempire.com/?page_id=110) and full [documentation here](http://www.powershellempire.com/?page_id=83).
+There's also a [quickstart here](https://github.com/BC-SECURITY/Empire/wiki/Quickstart).
```
Comment (vinnybod) [inline README.md]:
```suggestion
```

File: lib/common/empire.py [hunk 2]
```diff
@@ -15,7 +15,7 @@
 from builtins import str
 from builtins import range
 
-VERSION = "3.0.7 BC-Security Fork"
+VERSION = "3.1 BC-Security Fork"
```
Comment (vinnybod) [inline lib/common/empire.py]:
```suggestion
VERSION = "3.1.0 BC-Security Fork"
```

File: changelog [hunk 3]
```diff
@@ -1,3 +1,16 @@
+3/2/2020
+------------
+- Version 3.1 Master Release
```
Comment (vinnybod) [inline changelog]:
```suggestion
- Version 3.1.0 Master Release
```

File: README.md (hunk 1 above)
Comment (vinnybod) [reply README.md]: redundant since there's a Quickstart section below

File: README.md [hunk 4]
```diff
@@ -71,28 +86,26 @@ All image versions can be found at: https://hub.docker.com/r/bcsecurity/empire/
...
 ## To Do List
 
-* ~~Port code to work with Python 3~~
 * [Invoke-SocksProxy](https://github.com/p3nt4/Invoke-SocksProxy)
-* Function name randomization
-* ~~JA3/S signature randomization~~
-* ~~Multi-menu function calls~~
-* Function name aliasing
-* ~~Update to [Mimikatz 2.2.0](https://github.com/gentilkiwi/mimikatz)~~
+* Python 2.7 launchers calling back to Python 3.x
```
Comment (vinnybod) [inline README.md]: We should consider removing the Todo list here. Just create issues for things in the pipeline

File: VERSION [hunk 5]
```diff
@@ -1 +1 @@
-3.0.7
\ No newline at end of file
+3.1
```
Comment (vinnybod) [inline VERSION]:
```suggestion
3.1.0
```

File: README.md [hunk 6]
```diff
@@ -24,21 +25,35 @@
...
 
-In addition to the code conversion, there are some minor UI updates, a few new modules, and new functionality. The full list of changes can be reviewed in the changelog.
+Next, we have added the ability to randomize function names inside of Empire which we call Function Name Aliasing. This proof of concept is a huge step forward in evasion because it tests the ability to mask individual functions inside of Empire and further mask detection. Function name aliasing is currently only built for Invoke-Empire and Invoke-Mimikatz but will be expanded in the future.
 
+Please see our [Releases](https://github.com/BC-SECURITY/Empire/releases) or [Changelog](/changelog) page for detailed release notes.
 ## Install
+As of Empire 3.1.0, Empire only officially supports Python 3. If you still need Python 2 support, please use the [3.0.x branch](https://github.com/BC-SECURITY/Empire/tree/3.0.x) or releases. Also consider using our [Prebuilt Docker containers](#Docker) which use Python 3.
+
+__Note:__ Run ```./setup/reset.sh``` before starting Empire 3.1
```
Comment (vinnybod) [inline README.md]:
```suggestion
__Note:__ Run ```./setup/reset.sh``` before starting Empire 3.1 for the first time.
```

File: README.md (hunk 4 above)
Comment (Cx01N) [reply README.md]: Going to leave it for now.

**Pull request merged**